slides/01_course_introduction.html

```diff
@@ -404,6 +404,7 @@
 <li><a href="http://www.realtor.com/apartments/Goleta_CA">Realtor</a></li>
 <li><a href="http://www.trulia.com/for_rent/Goleta,CA">Trulia</a></li>
 <li><a href="http://www.padmapper.com/">PadMapper</a></li>
+<li><a href="https://santabarbara.craigslist.org//">Craigslist</a></li>
```

Comment:
Awesome! This file is auto generated from: https://github.com/scalableinternetservices/ucsb_website/blob/master/_slide_src/01_course_introduction.md

Could you edit that file instead? Thanks!